Added install script for Homebridge.

ludeeus opened the pull request:
As stated in the title and README this will install and configure Homebridge with Home Assistant addon.

ludeeus committed:
Added docs for Homebridge

ludeeus committed:
Added install script for Homebridge

ludeeus committed:
Corrected discord channel name

Landrash commented on package/opt/hassbian/suites/install_homebridge.sh:
```diff
@@ -0,0 +1,130 @@
+
+echo "Adding homebridge user, and creating config file..."
+sudo useradd -M --system homebridge
+sudo mkdir /home/homebridge
```
This should be able to be replaced by creating the home directory with useradd?

ludeeus committed:
Added option to expose to samba

- I samba is installed, users get option to expose `/home/homebridge/.homebridge` as `homebridge` user
- Corrected some language formatting
- Corrected discord channel in failed state

ludeeus committed:
Added installation instructions for homebridge.

Landrash reviewed (changes requested):
The creation of a folder should be handled by the correct flag with `useradd` instead of added manually. This to make sure that the proper permissions are applied to the home directory.

ludeeus committed:
Update install_homebridge.sh

ludeeus committed:
Merge branch 'dev' into patch-5

Landrash reviewed (approved):
Looks good! :cake:
